modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -73,22 +73,60 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					<aui:button cssClass="btn btn-primary btn-sm my-2" href="<%= cpContentHelper.getReplacementCommerceProductFriendlyURL(cpSku, themeDisplay) %>" value="replacement-product" />
 				</c:if>
 
-				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
-
-					<%
-					Format format = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
-					%>
+				<%
+				Format dateFormat = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
+				%>
 
+				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
 					<p class="my-2">
 						<span class="font-weight-semi-bold">
 							<%= LanguageUtil.get(request, "end-of-life") %>
 						</span>
 						<span>
-							<%= format.format(cpSku.getDiscontinuedDate()) %>
+							<%= dateFormat.format(cpSku.getDiscontinuedDate()) %>
 						</span>
 					</p>
 				</c:if>
 
+				<c:if test="<%= cpSku != null %>">
+					<div>
```

Comment:
Same here, what happens if I select another SKU?

Reply:
For multiple sku products there is a problem because, the jsp set cpSku to  cpContentHelper.getDefaultCPSku(cpCatalogEntry), and with multiple sku products is null. You showed me that javascript thing with ajax. But since the incoming quantity and the tooltip are a react's component, we can't move those inside that javascript file. For this behavior the incoming quantity is not showed in a multiple sku products

Reply:
Can't we handle it with events?

Reply:
I don't know what you mean with events, like click event or variable change event etc?

Reply:
Was this request addressed?